EULA.md

```diff
@@ -0,0 +1,291 @@
+*The Mass Open Cloud Alliance (MOC Alliance) has developed a production
+public cloud to provide domain researchers with predictable low-cost
+resources and facilitator support while enabling academic researchers
+and developers in the open source community to participate in close
+interactions between research, development, and production operations.
+Housed at the Boston University Hariri Institute for Computing and
+Computational Science & Engineering, the MOC Alliance is a unique
+collaborative effort between higher education, non-profit entities and
+industry (MOC Alliance Collaborators). For a full list of the MOC
+Alliance Collaborators, please visit*
+*[here](https://www.bu.edu/hic/centers-initiatives-labs/mocalliance/)*.
```

Comment:
Avoid links on the word "here" for documents that might be printed. Show the actual URL or section name instead:

```
...please visit <https://www.bu.edu/hic/centers-initiatives-labs/mocalliance/>.
```